Documentation/ceph-upgrade.md

```diff
@@ -399,6 +399,10 @@ To migrate the setting, follow these steps:
 
 ## Ceph Version Upgrades
 
+> **WARNING: There is a notice from Ceph for users upgrading to Ceph Pacific v16.2.6 or lower from
+> an earlier major version of Ceph. If you are upgrading to Ceph Pacific (v16), please upgrade to
+> v16.2.7 or higher if possible. See for more: https://github.com/rook/rook/issues/9185**
```

Comment:
Since v16.2.7 isn't available yet, how about if we also document here that they need to set the following before upgrading as you mentioned in the issue? 
```
ceph config set global bluestore_fsck_quick_fix_on_mount false
ceph config set osd bluestore_fsck_quick_fix_on_mount false
```

Reply:
I don't think they need to unless they have explicitly set them to `true` on their own. Those commands will only change what I wrote for my example, but it won't help a user who set them using `ceph config set osd.0 bluestore_fsck_quick_fix_on_mount true`.

Reply:
I like your latest suggestion on the issue to ensure they are unset. What if we just recommend they run the unset commands before upgrading to pacific 16.2.0-16.2.6?